src/main/perl/Fetch.pm

```diff
@@ -181,43 +173,48 @@ sub fetchProfile
 
     my $profile = $self->download("profile");
 
+    my $res;
     if (!defined($profile)) {
         $self->error("Failed to fetch profile $self->{PROFILE_URL}");
-        return undef;
-    }
-
-    return SUCCESS unless $profile;
-
-    local $SIG{__DIE__} = sub {
-        warn "Cleaning on die";
+    } elsif ("$profile" eq "0") {
+        # Have to do string comparison, because profile can be a FileReader or 0
+        $self->verbose("fetchProfile: nothing to do, ",
+                       "local cache is newer than profile (and no FORCE)");
```

Comment:
Strictly speaking this is really the profile that is older than the local cache! They can be the same date and in this case you will not reload it, isn'it? But I'm happy to leave with the message as it is...